docs/docs/FAQ/Mobile-App-FAQ.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 2
+---
+
+# Mobile App
+
+### What is the difference between the cloud icons on the mobile app?
+
+| Icon                               | Description                                                                                                                                     |
+| ---------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![cloud](/img/cloud.svg)           | Asset is only available in the cloud and was uploaded from some other device (like the web client) or was deleted from this device after upload |
+| ![cloud-cross](/img/cloud-off.svg) | Asset is only available locally and has not yet been backed up                                                                                  |
+| ![cloud-done](/img/cloud-done.svg) | Asset was uploaded from this device and is now backed up to the server; the original file is still on the device                      |
+
+
+### IOS app
+
+### No albums&photos seen by Immich on Iphone
+
+there are some reporting issue with iPhoneXR regarding the problem with scanning the album,we haven't yet to find the cause [see more info here](https://discord.com/channels/979116623879368755/1107718197655908412)
+
+:::note
+If you think you know what the solution to the problem might be, we'd be happy to discuss it on [Discord](https://discord.com/channels/979116623879368755/1071165397228855327)/[Github](https://github.com/immich-app/immich/discussions)
+:::
```

Comment:
I'm not sure if this is actually helpful. We don't have a solution for this issue, and it can point people in the wrong direction if the reason they don't see images is unrelated.

Reply:
I understand, I agree with that, in my opinion it might be worthwhile to create a place where the problems referred to the development team would be, this could prevent a lot of duplication in "issues".

Maybe I'll see if there is a bot that can look for duplicates in issues and creates a summary of the things that repeat themselves a lot or something like that (it would be interesting to know if there is something like that).